### ISSUE TITLE
Extra duplicated rows/columns in source png

### ISSUE THREAD

**Issue by srabouin** [umop-aplsdn/sc_decode#2]
It looks like sometimes a row or column repeats (not your script error, this is in the png prior to decrypt), I think those should be ignored. It seems to always be only one side so maybe it's based on rotation, I am not sure. If you look at the ui_spells, every card has a repeating row except the first and last column so it can be a good example to play with.

**Comment by knobse** [comment]
idk for sure but i think you are talking about the fact that they use CLAMP on those ? (example : http://write.flossmanuals.net/fluxus/texturing/static/Fluxus-texturing-04_texture_wrap-en.png)

**Comment by srabouin** [comment]
I think so, that must be it.. Not all of the sprites have it and not all sides...

**Comment by umop-aplsdn** [comment]
Yes, probably just some extra padding. These borders are hidden in game anyways. Can we close this?

**Comment by srabouin** [comment]
I don't know if I would close it, perhaps there is a way to remove them? like ignoring the pixels that are on the boundary?

**Comment by umop-aplsdn** [comment]
All images are cut and pasted according to the coordinates in the extracted sc files. There are some random misalignments, but the raw images seem fine. Try uncommenting line 334 (starts with "tmpRegion.save(") and check the extra dumped images (ui_spells_sprite_dbg_x_x.png).
